admin/site-admin/external-services.md

```diff
@@ -0,0 +1,40 @@
+## Integrate external services
+
+Stoplight allows you to integrate external services, such as remote code repositories, into your Stoplight organization. Stoplight supports the following integrations:
+
+- Bitbucket
+- More coming soon <!-- QUERY: which services are we going to support? -->
+
+Access **External Services** from the Stoplight dashboard by selecting:
+
+**admin** > **Site Admin** > **External Services**
+
+## Add a new service
+
+When you add a new service, project files automatically sync.
+
+<!-- QUERY: does the external repo need to have certain files/configs/setup? -->
+
+1. Select **Add Service**. This opens a modal.
+2. Provide the details of the new service, including:
+    1. **Provider**: this can be any of the supported integrations.
+    2. **Name**: choose a name to identify this service in the Stoplight interface.
+    3. **API URL**: 
+    4. **Username**: your username for the service.
+    5. **Password**: your password for the service.
+3. Select **SAVE**. Stoplight returns you to the **External Services** list. You can view and edit your new integration.
+
```

Comment:
We're adding a Test Connection Button to this modal right now.

Reply:
Kewl. Will pick that up next sprint. Suspect I simply need a bunch of small stories each sprint to review & pick up changes from various dev areas. Have added to backlog here: https://stoplightio.atlassian.net/browse/DOC-73
We can duplicate that task for every sprint if needed.